extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -36,7 +42,8 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 			},
 			slotName = 'INCONTENT_BOXAD_1',
 			startPosition = placeHolderElement.offsetTop,
-			swapRecirculationAndAd;
+			swapRecirculationAndAd,
+			btRec = babDetection.isBlocking() && wadRecRunner.isEnabled('bt');
```

Comment:
We can initialise this variable with `null` value, because FMR can be showed or not.